pkg/machine/qemu/machine.go

```diff
@@ -199,6 +207,15 @@ func (v *MachineVM) Init(opts machine.InitOptions) error {
 			return errors.Errorf("error resizing image: %q", err)
 		}
 	}
+	// If the user provides an ignition file, we need to
+	// copy it into the conf dir
+	if len(opts.IgnitionPath) > 0 {
+		inputIgnition, err := ioutil.ReadFile(opts.IgnitionPath)
+		if err != nil {
+			return err
+		}
+		return ioutil.WriteFile(v.IgnitionFilePath, inputIgnition, 0644)
```

Comment:
Should mode be matched to `opts.IgnitionPath` rather hard coded to 0644?

Reply:
i'm matching to what i normally create the ignition files as ...